### DISCUSSION TITLE
Difference between baseline and since?

### DISCUSSION THREAD

**Discussion by theagilepilot** [stryker-mutator/stryker-net#2860] [Q&A] [answered]
I am looking to use Stryker.NET in a CI pipeline. Is there any advantage to using "with-baseline" over "since" (compared to the same differences being tested)?

**Comment by rouke-broersma** [comment, marked as answer]
Baseline aims to provide a full report even though only a partial test is performed. It does so by reusing a previous result and modifying this with the incremental result. Since does not do this, so since only shows you mutants in changed code, or for mutants of which the covering tests have changed.

**Reply by theagilepilot** [reply]
Thanks for clarifying.
